### DISCUSSION TITLE
Put 5 inch dsi screen display in French

### DISCUSSION THREAD

**Discussion by patrickpatoche02** [nebhead/PiFire#193] [General]
Hello, I am French so excuse the pronunciation.
 first of all this project is great, I have an old model Pit Boss 850 and I want to replace the old electronics with the superb PIFIRE.
I did the electronic editing with a Rasbperry 3+ and a 5'' 800x480 DSI screen.
I would like to try to put the text displayed on the screen in French, is this possible?
Sincerely
Patrick

**Comment by nebhead** [comment]
Hi Patrick!  Thanks for using PiFire!  Localization has definitely been something on my to-do list for a long time.  I would need some help with this with native speakers to provide me with the language changes, but it's something that is possible in the future.

**Reply by patrickpatoche02** [reply]
good evening, if you need a volunteer to translate English to French. I am OK

**Reply by nebhead** [reply]
Thanks!  I'll let you know when I get to the point where we can do that.  I think for displays, in the future, it should be easy with a new system for configuring the display.  For the webui, it might be a bit more challenging, but I'll have to do some research on if there are easy ways to accomplish this.

**Reply by patrickpatoche02** [reply]
Good morning. when you are near, tell me.
I would be willing to help you.
Sincerely
Patrick
